MigrationAndImplementationGuide1-4.md

```diff
@@ -225,7 +334,7 @@ Most of the knowledge graph will remain the same in this example. Only one of th
     "predicate": "treats",
     "attributes": [
         {
-            "attribute_type_id": "support graph",
+            "attribute_type_id": "infores:support_graphs",
```

Comment:
is this supposed to be "biolink:support_graphs", from [here](https://github.com/biolink/biolink-model/blob/35925815e6ef47c0e4a0e28417452be486c1c18f/biolink-model.yaml#L672)?

Reply:
Fixed. Thank you.

There's noting in the specification that necessarily says that it has to be a list, but I think expected practice is a list as opposed to a single value. Unless this was further specified in the biolink model.